### DISCUSSION TITLE
From Radon2D To Image

### DISCUSSION THREAD

**Discussion by fatihtalu** [PyLops/pylops#344] [Q&A]
I have a 2D binary image.
I get the sinogram of this with RADON2D.
Next I want to draw the line corresponding to the maximum value cell in the sinogram over my binary image in the spatial domain.

I get the theta and px values of the max valued cell in the sinogram, but I don't know how to draw the line on the image using these values.

Thank you

[Code.pdf](https://github.com/PyLops/pylops/files/8190114/Code.pdf)

**Comment by mrava87** [comment]
Sorry I did see this, see associated GitHub issue
